systems/plants/test/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ if (eigen3_FOUND)
   add_executable(urdfKinTest urdfKinTest.cpp)
   include_directories( .. )
   target_link_libraries(urdfKinTest drakeRBM)
+  add_executable(rbmConstructorDestructorTest rbmConstructorDestructorTest.cpp)
+  include_directories( .. )
+  target_link_libraries(rbmConstructorDestructorTest drakeRBM)
```

Comment:
adding an add_test would give it coverage.  but i’d probably vote zap it.  we have enough other valgrind-able examples (e.g. urdfKinTest)

On Mar 30, 2015, at 2:51 PM, Andres Valenzuela <notifications@github.com> wrote:

> In systems/plants/test/CMakeLists.txt:
> 
> > @@ -6,6 +6,9 @@ if (eigen3_FOUND)
> >    add_executable(urdfKinTest urdfKinTest.cpp)
> >    include_directories( .. )
> >    target_link_libraries(urdfKinTest drakeRBM)
> > +  add_executable(rbmConstructorDestructorTest rbmConstructorDestructorTest.cpp)
> > +  include_directories( .. )
> > +  target_link_libraries(rbmConstructorDestructorTest drakeRBM)
> I didn't add_test it because the intended use case is with valgrind. Should I zap it then?
> 
> —
> Reply to this email directly or view it on GitHub.
>